recipes-kernel/framegrabber/framegrabber_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Framegrabber kernel module"
+DESCRIPTION = "Kernel module to load video frames captured from ARP cameras in to memory"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+PV = "1.0+git${SRCREV}"
+PR = "r0"
+
+inherit module
+
+SRCREV = "e6ac6f58abee88d04d7c92785c2c9a6c3868aa48"
+
+SRC_URI = "git://github.com/Pelagicore/framegrabber.git;protocol=https;branch=master"
+
+S = "${WORKDIR}/git"
+
+# The inherit of module.bbclass will automatically name module packages with
```

Comment:
why is this needed?

Reply:
So it fetches the sources in the S directory to compile and install

Reply:
I've meant the comment.

Reply:
Its not needed. The comment was there from where I fetched this recipe. So I didn't removed this. Do you think this is not a useful comment?

Reply:
I thought you were the author; it's not important

Reply:
Tbh, I kind of agree, at the very least the comment is in the wrong location in the file imho. It's a dangling comment.